aleph.env.tmpl

```diff
@@ -125,3 +125,6 @@ SENTRY_DSN=
 
 # Sentry environment description, see https://docs.sentry.io/platforms/python/configuration/environments/
 SENTRY_ENVIRONMENT=
+
+# Number of threads to use in servicelayer workers, 0 = disabled.
```

Comment:
Maybe provide a bit more context here. I'd like to see some information on why we have this set to zero, or a link to docs that explain how to determine the number of servicelayer workers?

Reply:
Agreed, will improve this.